doc/GuideForCommitters.md

```diff
@@ -91,21 +91,21 @@ build, add a comment to the pull request that follows the syntax:
    Jenkins build [ all | {platform-list} ]
    ```
 Where `{platform-list}` is a comma-separated list of platforms chosen from the
-following tags:
-
-| Tag      | Platform |
-| :----    | :------- |
-| zlinux   | 64-bit Linux on Z |
-| zos      | 64-bit zOS |
-| plinux   | 64-bit Power Linux LE |
-| aix      | 64-bit AIX |
-| x32linux | 32-bit Linux |
-| xlinux   | 64-bit Linux |
-| xcrlinux | 64-bit Compressed References Linux |
-| win      | 64-bit Windows |
-| osx      | 64-bit macOS |
-| arm      | 32-bit AArch32 (cross-compile build only) |
-| aarch64  | 64-bit AArch64 (cross-compile build only) |
+following aliases:
```

Comment:
I think this would read better as "following names and aliases" with the "Spec Name" column first; the "Alias" column could be second or last.

Reply:
Updated.